extensions/sql-database-projects/src/models/project.ts

```diff
@@ -964,7 +964,7 @@ export class Project implements ISqlProject {
 		const toDeleteFolders: FileProjectEntry[] = this._files.filter(x => x.fsUri.fsPath.startsWith(entry.fsUri.fsPath) && x.type === EntryType.Folder);
 
 		await Promise.all(toDeleteFiles.map(x => fs.unlink(x.fsUri.fsPath)));
-		await Promise.all(toDeleteFolders.map(x => fs.rmdir(x.fsUri.fsPath, { recursive: true })));
+		await Promise.all(toDeleteFolders.map(x => fs.rm(x.fsUri.fsPath, { recursive: true })));
```

Comment:
does `force: true` also need to be added in the options, like mentioned in the deprecated comment? 
![image](https://user-images.githubusercontent.com/31145923/205392716-621e0c09-cc04-4c01-abab-1baa7e9d1260.png)

Reply:
I will add that as well based on the comment. But I didn't add that before because when I tested the changes, they were working as expected. But just to make sure I don't miss anything, I will add the force flag.